test/unit/models/publishes_to_publishing_api_test.rb

```diff
@@ -2,85 +2,68 @@
 require 'sidekiq/testing'
 
 class PublishesToPublishingApiTest < ActiveSupport::TestCase
-  # publish_to_publishing_api runs on after_commit hook, so we need to disable
-  # transactions for this test to allow it to run.
-  self.use_transactional_fixtures = false
+  class TestObj
+    include ActiveModel::Validations
+    include ActiveModel::Validations::Callbacks
 
-  setup do
-    DatabaseCleaner.clean_with :truncation
-    stub_any_publishing_api_call
-  end
+    def self.after_commit
+    end
 
-  teardown do
-    DatabaseCleaner.clean_with :truncation
-  end
+    def persisted?
+      true
+    end
 
-  test "generating a content_id" do
-    test_uuid = SecureRandom.uuid
-    SecureRandom.stubs(uuid: test_uuid)
-    organisation = create(:organisation)
-
-    assert organisation.valid?
-    assert_equal test_uuid, organisation.content_id
+    def search_link
+      "test_link"
+    end
   end
 
-  test "create publishes to Publishing API if not disallowed" do
-    organisation = build(:organisation)
-    Whitehall::PublishingApi.expects(:publish_async).with(organisation)
-    organisation.save
+  def include_module(object)
+    class << object
+      include PublishesToPublishingApi
+    end
+    object
   end
 
-  test "update publishes to Publishing API if not disallowed" do
-    organisation = create(:organisation)
-    Whitehall::PublishingApi.expects(:publish_async).with(organisation)
-    organisation.update_attribute(:name, 'Edited org')
+  setup do
+    TestObj.stubs(:after_commit).with(:publish_to_publishing_api,
```

Comment:
Can you add spaces around hash braces per our style guide.

Also - here and elsewhere - for multiple method arguments can you place each argument on a separate line, including the first argument and closing paren. i.e.

```ruby
with(
  :blah,
  :bleh
)
```
Aside: it seems this project's linter version is way out of date otherwise these things would have been picked up.